cachi2/interface/cli.py

```diff
@@ -179,21 +202,58 @@ def combine_flags() -> list[str]:
         return flags
 
     parsed_packages = tuple(chain.from_iterable(map(parse_packages, package)))
-
-    try:
-        request = Request(
-            source_dir=source,
-            output_dir=output,
-            packages=parsed_packages,
-            flags=combine_flags(),
-        )
-    except pydantic.ValidationError as e:
-        print_error(str(e))
-        raise typer.Exit(1)
+    request = Request(
+        source_dir=source,
+        output_dir=output,
+        packages=parsed_packages,
+        flags=combine_flags(),
+    )
 
     request_output = gomod.fetch_gomod_source(request)
 
     request.output_dir.mkdir(parents=True, exist_ok=True)
     request.output_dir.joinpath("output.json").write_text(request_output.json())
 
     log.info(r"All dependencies fetched successfully \o/")
+
+
+@app.command()
+@friendly_errors
+def generate_env(
+    from_output_dir: Path = Option(
```

Comment:
Do you think changin this to an Argument would make it clearer for the users?
```
cachi2 generate-env ./my/random/output/dir
```

Reply:
Not sure if clearer, IMO the name of the param describes the meaning of the path. But it might be nicer to use

Reply:
Changed into a *required* argument, maybe that's better?